server/worker/apple_mdm.go

```diff
@@ -655,6 +663,7 @@ func QueueAppleMDMJob(
 	teamID *uint,
 	enrollReference string,
 	useWorkerDeviceRelease bool,
+	fromMDMMigration bool,
```

Comment:
Not sure if we are reaching the point of where some of these arguments could be grouped in a struct to make it more clear.

Will approve anyways.